css/styles.css

```diff
@@ -0,0 +1,526 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.advertisementSection {
+  width: 100%;
+  box-sizing: border-box;
+  padding: 50px;
+  background-color: #888;
+}
+
+.advertisementSection__title1 {
+  font-size: 1.25rem;
+}
+
+.advertisementSection__title2 {
+  font-size: 2.5rem;
+  display: block;
+}
+
+.advertisementSection__link {
+  width: 150px;
+  line-height: 2rem;
+  font-size: 2rem;
+  background-color: #666;
+  border-radius: 10px;
+  padding: 20px;
+  vertical-align: top;
+  position: relative;
+  top: 50px;
+  text-decoration: none;
+  color: black;
+}
+
+.advertisementSection__text {
+  font-size: 1.25rem;
+  line-height: 2rem;
+}
+
+.advertisementSection__img {
+  float: right;
+  height: 350px;
+  margin-top: -100px;
+}
+
+.advertisementSection::after {
+  content: "";
+  display: block;
+  clear: both;
+}
+
+.article {
+  width: 37rem;
+  background-color: #777;
+  display: flex;
+  flex-direction: column;
+}
+
+.article__articleTitlerP {
+  margin-left: 2rem;
+}
+
+.article__content {
+  margin: 1rem;
+}
+
+.article__tag {
+  color: white;
+}
+
+.article__thumbAuthor {
+  margin-top: 2rem;
+  margin-left: 2rem;
+}
+
+.articleAuthorP {
+  color: white;
+  text-shadow: 1px 1px 1px rgba(0, 0, 0, 0.25);
+  font-weight: bold;
+  width: max-content;
+  margin: 0;
+}
+
+.articleHeader {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.articleTags_hash {
+  color: #222;
+  font-weight: bold;
+  margin-left: 2rem;
+}
+
+.articleTags_text {
+  color: white;
+  font-weight: bold;
+}
+
+.articleTags_number {
+  font-weight: bold;
+  color: white;
+}
+
+.articleTitleP {
+  color: white;
+  background-color: #333;
+  font-size: 3rem;
+  font-weight: bold;
+}
+
+.articleTitleP_wrote {
+  font-style: italic;
+}
+
+.body {
+  padding: 0;
+  margin: 0;
+  width: 100%;
+  background-color: #999;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.body__header {
+  margin: 0;
+}
+
+.header {
+  padding: 0;
+}
+
+.header__nav {
+  margin: 10px;
+}
+
+.main {
+  padding: 0;
+}
+
+.main__advertisementSection {
+  margin-top: 100px;
+}
+
+.main__article {
+  float: left;
+}
+
+.main__personOfWeek {
+  margin: auto;
+}
+
+.main__secondaryArticlesSection {
+  margin: 5% 3%;
+}
+
+.main__title {
+  margin: auto;
+}
```

Comment:
This selector has the same style definition of the selectors on line 150, group selectors with the same style definition.